CHANGELOG.md

```diff
@@ -12,5 +12,6 @@ All notable, unreleased changes to this project will be documented in this file.
 
 ### Other changes
 - Improved page search with search vectors. Pages can now be searched by slug, title, content, attribute values, and page type information.
+- When installing apps that have the same identifier in manifest as an app that is already installed in Saleor, `AppErrorCode.UNIQUE` error code will be now returned instead of `AppErrorCode.INVALID` (backported to 3.22.1)
```

Comment:
Unnecessary to include the change in the 3.23.0 changelogs as it will compare against 3.22.x:

```suggestion
```